k6/load.js

```diff
@@ -0,0 +1,51 @@
+import { check, sleep } from 'k6';
+import exec from 'k6/execution';
+import http from 'k6/http';
+
+export const options = {
+	//vus and duration can be changed depending on what simulation needed running
+	scenarios: {
+    	contacts: {
+      	executor: 'constant-vus',
+      	vus: 100,
+      	duration: '30m',
+    	},
+  	},
```

Comment:
Indentation is messed up here, probably because this folder isn't an npm project so we're not running linting / prettifier on push. Maybe we can do something about that...